projects/schematics/src/shared/utils/lib-utils.ts

```diff
@@ -452,9 +417,33 @@ function addFeatureModule<T extends LibraryOptions>(
   };
 }
 
+/**
+ * Checks if the given feature module is already being
```

Comment:
Maybe something like "Checks if the given feature module is already imported in a wrapper module"

Reply:
Nice, thanks :)